packages/management/tkr-service/bundle/config/upstream/tkr-status-controller-deployment.yaml

```diff
@@ -0,0 +1,51 @@
+#@ load("@ytt:data", "data")
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: tkr-status-controller
+  name: tkr-status-controller-manager
+  namespace: #@ data.values.namespace
+  annotations:
+    kapp.k14s.io/change-rule.0: "upsert after upserting tkr-service.tanzu.vmware.com/ClusterRoleBinding"
+    kapp.k14s.io/change-rule.1: "delete before deleting tkr-service.tanzu.vmware.com/ClusterRoleBinding"
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: tkr-status-controller
+  template:
+    metadata:
+      labels:
+        app: tkr-status-controller
+    spec:
+      containers:
+        - image: tkr-status-controller:latest
+          imagePullPolicy: Always
```

Comment:
question: 
Are we always pulling the image? Is this overridden by an overlay or something when building the package?

Reply:
Yes, we are not overriding the overlay.

Reply:
fixed: chnaged it to `IfNotPresent`